packages/@aws-cdk/aws-eks/test/cluster.test.ts

```diff
@@ -656,7 +656,7 @@ describe('cluster', () => {
     Template.fromStack(stack).hasResourceProperties('Custom::AWSCDK-EKS-Cluster', {
       Config: {
         roleArn: { 'Fn::GetAtt': ['ClusterRoleFA261979', 'Arn'] },
-        version: '1.21',
+        version: '1.25',
```

Comment:
shouldn't this just be `CLUSTER_VERSION`?

Reply:
same comment to all the other places where `1.25` is hardcoded

Reply:
@kaizencc I guess my github notification got buried somewhere. I didn't see this until today. 😢 

Yes. There was lots of hardcoding. I'll address that too. Thanks ❤️